code/game/machinery/exp_cloner.dm

```diff
@@ -0,0 +1,293 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+/obj/machinery/clonepod/experimental
+	name = "experimental cloning pod"
+	desc = "An ancient cloning pod, built when cloning technology wasn't perfected yet."
```

Comment:
Wasn't the station's cloning technology considered experimental back in the day?

Reply:
That's a good point yeah, maybe change this to something like "that predates even the experimental cloner that comes equipped with some nanotrasen stations."